web/views.py

```
@@ -4700,6 +4702,286 @@
 
 
 @login_required
+@require_POST
+def update_student_attendance(request):
+    if not request.headers.get("X-Requested-With") == "XMLHttpRequest":
```

Ruff (0.8.2)

Use `request.headers.get("X-Requested-With") != "XMLHttpRequest"` instead of `not request.headers.get("X-Requested-With") == "XMLHttpRequest"`

Replace with `!=` operator

(SIM201)